java/java.lsp.server/build.xml

```diff
@@ -96,6 +96,8 @@
         </exec>
         <exec executable="${build.dir}/vsce/node_modules/vsce/out/vsce" failonerror="true" dir="${basedir}/vscode">
             <arg value="package" />
+            <arg value="--baseImagesUrl" />
+            <arg value="https://github.com/apache/netbeans/raw/master/java/java.lsp.server/vscode" />
```

Comment:
Using `master` means that such images can never be removed. Wouldn't it be more proper to obtain the actual commit ID and use it in the URL?

Reply:
Fixed as requested.